actions/rssiActions.js

```diff
@@ -152,7 +156,8 @@ export function open(serialPort) {
                     throttleUpdates = false;
                     dispatch(rssiData());
                 });
-            });
+            })
+                .on('error', console.log);
```

Comment:
How about `logger.error` instead?

Reply:
It was on purpose on console.log, I can remove it though.